plone/namedfile/utils/jpeg_utils.py

```diff
@@ -29,17 +28,18 @@ def process_jpeg(data):
                 if (ord(b) >= 0xC0 and ord(b) <= 0xC3):
                     jpeg.read(3)
                     h, w = struct.unpack('>HH', jpeg.read(4))
+                    content_type = 'image/jpeg'
```

Comment:
I assumed that when this method is failing to extract width and height info, it's broken anyways.
But that's false.
I'll fix and ask you to review then.